recipe/meta.yaml

```diff
@@ -85,7 +85,7 @@ test:
     - commands_to_test.bat           # [win]
     - commands_to_test.sh            # [unix]
   downstreams:
-    - conda-smithy  # [python_impl == "cpython"]
+    - conda-smithy  # [python_impl == "cpython" and py<311]
```

Comment:
Instead of dropping this selector, maybe we can just bump it when `conda-smithy` is installable on Python 3.11. That way things are ready to go for the next Python migration.

xref: https://github.com/conda-forge/conda-feedstock/pull/180